### PR TITLE
Style create form

### PR DESCRIPTION
**Pull request by Gpx**
Before 
Screen Shot 2019-07-16 at 10 27 07 AM

After 
Screen Shot 2019-07-16 at 12 05 23 PM

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,10 +1,9 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: #1D2C3C;
```
Comment (rogerclotet) [inline src/index.css]: Isn't this color in a variable?

Comment (atoro-travelperk) [reply src/index.css]: Can we take the color from a tsx file to put it in a css/styled ?

Comment (Gpx) [reply src/index.css]: Yeah, it's madrugada but I can't import JS variables in a CSS files

Comment (Gpx) [reply src/index.css]: > Can we take the color from a tsx file to put it in a css/styled ?

What do you mean?

Comment (atoro-travelperk) [reply src/index.css]: > Yeah, it's madrugada but I can't import JS variables in a CSS files
This is the answer to my quetion. Thanks @Gpx !

**Pull request merged**